### PR TITLE
suggest models in the code editor

### LINKED ISSUE #3384: Suggest models in autocomplete
**Is your feature request related to a problem? Please describe.**

The autocomplete should also show modelnames.

**Describe the solution you'd like**

Models are shown in the drop-down of the autocomplete.

**Describe alternatives you've considered**

Not sure. Models could benefit from a slightly different color-formatting, as opposed to sources (?) 

**Additional context**

The autocomplete only suggests `tradesdownload` (a source). It should suggest `types` (a model) as well.
image

fwiw - the entire expression works great, it is just the auto-complete :) 

image

My apologies if this is already logged, or part of https://github.com/rilldata/rill/issues/265

Found in `version 0.36.7 – c13ed366`.

### PR DESCRIPTION
**Pull request by briangregoryholmes**
Closes #3384

Like the idea of coloring sources and models differently, but this PR does not address that feature.

### CODE REVIEW

**Comment by ericpgreen2** [comment]
I started on QA, but I hit this error in my browser console, and it crashed the app:
image

I have a `select * from adbids` model, and I'm able to trigger the error when I refresh the page.

File: web-common/src/features/models/selectors.ts [hunk 1]
```diff
@@ -31,6 +37,26 @@ export function useModel(instanceId: string, name: string) {
...
+    if (!allModels.data?.length) {
+      set([]);
+      return;
+    }
+
+    derived(
+      allModels.data.map((r) =>
+        createTableColumnsWithName(queryClient, instanceId, r.meta.name.name)
+      ),
+      (sourceColumnResponses) =>
```
Comment (ericpgreen2) [inline web-common/src/features/models/selectors.ts:54]: Ah, this tipped me off to where you got this code :). Nice approach to just mirror what we have for sources. But: `sourceColumnResponses` -> `modelColumnResponses`

**Comment by briangregoryholmes** [comment]
@ericpgreen2 I missed that `schema` was technically undefined until it hit the reactive code block that populated the source columns.

I moved the empty object assignment to the variable declaration, which resolved the issue on my end.

File: web-common/src/features/models/selectors.ts (hunk 1 above)
Comment (briangregoryholmes) [reply web-common/src/features/models/selectors.ts]: Renamed

**Review by ericpgreen2 — APPROVED** [review]
Works great!

**Pull request merged**